### ISSUE TITLE
$mdMedia documentation

### ISSUE THREAD

**Issue by STOpandthink** [angular/material#6134]
https://material.angularjs.org/latest/api/service/$mdMedia
Am I just confused, or is it incorrectly written. I'm doing this:

```
$scope.$watch(function() { return $mdMedia('gt-sm'); }, function(big) {
  console.log(big);
});
```

I expect that it will print "false" when the screen width is > 960px, and print true otherwise. This seems what the documentation is saying. But instead it uses the threshold of 600px, which is actually consistent with all the other things, e.g. layout-align-gt-sm. So my guess is that the documentation is phrased incorrectly or poorly.

**Comment by topherfangio** [comment]
This may have been updated in our 1.0.0 release. Can you double-check that this is still broken, and if so, provide a Codepen which demonstrates it so that we can have a good test case?

**Comment by STOpandthink** [comment]
It's fixed in 1.0, thanks!
